packages/extension_google_sign_in_as_googleapis_auth/example/android/app/build.gradle

```diff
@@ -60,5 +61,7 @@ flutter {
 dependencies {
     implementation 'com.google.android.gms:play-services-auth:16.0.1'
     testImplementation'junit:junit:4.12'
-    testImplementation 'org.mockito:mockito-core:2.17.0'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
```

Comment:
We don't run Espresso tests in this repo currently, and I didn't see any in this PR, so I wouldn't expect us to need this.